program_info/qt-installer/packages/com.microsoft.VCRedist/meta/installscript.js

```diff
@@ -0,0 +1,31 @@
+function Component() { }
+
+function vcRedistExeName(architecture) {
+	switch (systemInfo.currentCpuArchitecture()) {
+		case "x86_64":
+			return "vc_redist.x64.exe";
+		case "arm64":
+			return "vc_redist.arm64.exe";
+		default:
+			throw new Error("You CPU Architecture is not supported! Womp womp");
+	}
+}
+
+function handleVCRedist() {
+	const exeName = vcRedistExeName(architecture);
+	const exePath = "@TargetDir@/" + exeName;
+
+	component.addElevatedOperation("Execute", vcpath, "/quiet", "/norestart");
+	component.addOperation("Delete", vcpath);
+}
+
+Component.prototype.createOperations = function() {
+	try {
+		// Call default impl first
+		component.createOperations();
+
+		handleVCRedist();
+	} catch(error) {
+		console.log(error);
+	}
+}
```

Comment:
I have no idea if this is possible, but this is a bit of a dated way of doing things in JavaScript

AFAIK it is identical to

```js
class Component {
    static createOperations() {
        // ...
    }
}
```

and basically the same as
```js
const Component = {
   createOperations() {
       // ...
   }
};
```

Is this an old ES version or is the engine just very fussy?